### PR TITLE
OSSM-6086: Formatting issue in federation gateway parameters table

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): OCP 4.15, 4.14

Issue: https://issues.redhat.com/browse/OSSM-6086

Link to docs preview: https://73013--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-federation.html#understanding-federation-gateways

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-federation-config-smcp.adoc [hunk 1]
```diff
@@ -120,15 +120,15 @@ To avoid naming conflicts between meshes, you must create separate egress and in
 |spec:
   gateways:
     additionalEgress:
-      <egressName>:
+      <egress_name>:
```
Comment (eoinfennessy) [inline modules/ossm-federation-config-smcp.adoc:123]: If we want to adopt a convention for using snake_case for anything contained within angle brackets, should [everything on this page](https://73013--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-federation) be changed to follow this convention?

**Review by eoinfennessy — CHANGES_REQUESTED** [review]
The table looks good. I added two comments about the style convention changes.

File: modules/ossm-federation-config-smcp.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-federation-config-smcp.adoc:123]: @eoinfennessy yes, that's the goal per the official style guide.

File: modules/ossm-federation-config-smcp.adoc [hunk 2]
```diff
@@ -17,8 +17,8 @@ In the following example, the administrator for the `red-mesh` is configuring th
 apiVersion: maistra.io/v2
 kind: ServiceMeshControlPlane
 metadata:
-  name: red-mesh
-  namespace: red-mesh-system
+  name: red_mesh
+  namespace: red_mesh_system
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-federation-config-smcp.adoc:21]: @eoinfennessy yes, thank you for clarifying this for me. This is the feedback I needed. I wasn't sure where all the places were to make the changes. I'll give it another pass and contact you again. Thanks!

[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-federation-config-smcp.adoc]: updated

File: modules/ossm-federation-config-smcp.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-federation-config-smcp.adoc]: created OSSM-5966 to capture the work on other pages. using this PR only for changes on this PR as this was technically scope creep that delayed the change associated with the intent of the initial story.  :)

**Review by eoinfennessy — APPROVED** [review]

**Review by pbajjuri20 — APPROVED** [review]
LGTM..!

**Comment by rh-tokeefe** [comment]
label peer-review-needed

**Comment by rh-tokeefe** [comment]
This PR essentially updates formatting for content that was previously reviewed and approved.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**